experiment/scheduler.py

```diff
@@ -715,8 +715,14 @@ def render_startup_script_template(instance_name: str, fuzzer: str,
     """Render the startup script using the template and the parameters
     provided and return the result."""
     fuzzer_config = fuzzer_config_utils.get_by_variant_name(fuzzer)
+    local_experiment = experiment_utils.is_local_experiment()
```

Comment:
I want to minimize the number of times we branch on is_local_experiment.
Let's make docker_registry mandatory for every experiment.

Reply:
This reason sounds reasonable.

Reply:
By the way, this one can be removed. Actually I added the `dcoker_registry` here is because of unit tests.